src/routes/privateRoute/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { Navigate, Outlet } from "react-router-dom";
+import { urlFormat, useAuth } from "@/utils";
+import { Pages } from "@/pages";
+
+const PrivateRoute = () => {
+  const auth = useAuth();
+  return auth ? <Outlet /> : <Navigate to={urlFormat(Pages.SignIn)} />;
```

Comment:
```suggestion
  const user = useAuth();
  return user ? <Outlet /> : <Navigate to={urlFormat(Pages.SignIn)} />;
```

Reply:
I prefer user because I think it’s more clear to know what is returning `useAuth`

Reply:
Perfect, updated!